lisa/features/gpu.py

```diff
@@ -115,21 +115,43 @@ def _install_cuda_driver(self, version: str) -> None:
             )
         elif isinstance(self._node.os, Ubuntu):
             release = re.sub("[^0-9]+", "", os_information.release)
-            cuda_repo_pkg = f"cuda-repo-ubuntu{release}_{version}_amd64.deb"
-            cuda_repo = (
-                "http://developer.download.nvidia.com/compute/"
-                f"cuda/repos/ubuntu{release}/x86_64/{cuda_repo_pkg}"
-            )
             # Public CUDA GPG key is needed to be installed for Ubuntu
             self._node.execute(
                 "apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/"
                 f"cuda/repos/ubuntu{release}/x86_64/7fa2af80.pub",
                 sudo=True,
             )
-            # download and install the cuda driver package from the repo
-            self._node.os._install_package_from_url(
-                f"{cuda_repo}", package_name="cuda-drivers.deb", signed=False
-            )
+            if "2004" == release:
```

Comment:
Does it work with 2004 only? For this kind of pattern, we can assume the latest format work with future versions. So the logic should be like
```python
if "1804" == release:
   # logic of 1804
else:
   # logic of 2004
```